app/javascript/packs/acuant_document_capture.js

```diff
@@ -0,0 +1,5 @@
+import { loadAndInitializeAcuantSdk } from '../app/acuant/document_capture';
+
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Just realized .... since `loadAndInitializeAcuantSdk` is a no-args function....we can just do:

```js
document.addEventListener('DOMContentLoaded', loadAndInitializeAcuantSdk)

```